### ISSUE TITLE
Settings jumps around between radio buttons and checkboxes

### ISSUE THREAD

**Issue by upstreamplugin** [upstreamplugin/UpStream#386]
The UpStream settings are very inconsistent between radio buttons and checkboxes

![screen shot 2018-02-23 at 3 10 45 pm](https://user-images.githubusercontent.com/26938537/36614790-d6bce420-18ab-11e8-9007-a1063dd63495.png)

**Comment by deenison** [comment]
Maybe we should revisit the whole settings page layout

**Comment by upstreamplugin** [comment]
Yes, although thinking about it, we should probably tag this "Gutenberg". When that arrives, lots of settings may move around
